src/AdminService.Data.Provider/IDataProvider.cs

```diff
@@ -0,0 +1,10 @@
+﻿using LT.DigitalOffice.Kernel.Attributes;
+using LT.DigitalOffice.Kernel.Enums;
+
+namespace AdminService.Data.Provider
+{
+  [AutoInject(InjectType.Scoped)]
+  public interface IDataProvider

```

Comment:
maybe IDataProvider should inherit IBaseDataProvider

Reply:
add